clients/vscode/src/StatusBarItem.ts

```diff
@@ -181,11 +160,6 @@ export class StatusBarItem {
     this.item.backgroundColor = backgroundColorWarning;
     this.item.text = `${iconDisconnected} ${label}`;
     this.item.tooltip = "Cannot connect to Tabby Server. Click to open settings.";
-    this.item.command = {
-      title: "",
-      command: "tabby.applyCallback",
-      arguments: [() => this.issues.showHelpMessage("connectionFailed")],
-    };
```

Comment:
There will be no button to show help message after this change. Is it expected?

Reply:
it'll open settings page and show `Disconnected` in palette

![image](https://github.com/user-attachments/assets/5084da4c-03c3-4107-91eb-f69ce22c3476)